### ISSUE TITLE
website search console shows many pages 404 not found

### ISSUE THREAD

**Issue by masinter** [Interlisp/medley#1588]
A lot of the 404 not found errors reported were pointers to URLs with _print in them.
There is a global option in hugo to disable the "feature" of adding a link in the URL to "generate a printed version of this section". I don't think we have any such pages and if we did, they wouldn't be in markdown.
So if you turn that option off (whose name escapes me), you'll get rid of the errors that are preventing the page that points to 404's from being indexed by Google.

**Comment by pamoroso** [comment]
@stumbo is the Hugo option turned on? Does the issue still occur?

**Comment by stumbo** [comment]
No,  this issue no longer occurs.  We disabled the print section option in 2024 (from `hugo.yaml`):

```
# Comment out if you don't want the "print entire section" link enabled.
# [outputs]
# section = ["HTML", "print"]
```

When enabled this creates URLs with `_print` in them.
